src/components/Header/Header.styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const Title = styled.div`   
```

Comment:
Seems like there is a gap between the top and sides of the header component

<img width="960" alt="image" src="https://github.com/CommunityFocus/cf-frontend/assets/45009203/7bbb9de2-0ea9-48b5-b8a9-074cc6e9cccc">